### ISSUE TITLE
Running test case `test_vfio` on X86_64 desktop failed

### ISSUE THREAD

**Issue by michael2012z** [cloud-hypervisor/cloud-hypervisor#5374]
**Describe the bug**
Running test case `test_vfio` on X86_64 Ubuntu desktop failed

**To Reproduce**

I was trying to run the integration test `test_vfio` on my X86_64 Ubuntu desktop.
I removed the `ignore` annotation of the test (https://github.com/cloud-hypervisor/cloud-hypervisor/blob/41880919a0a4eb0e4fe4007337ac886128906c43/tests/integration.rs#L3720). This test case was ignored for #4324 in CI, but I think it should work for my local test:

`./scripts/dev_cli.sh tests --integration -- --test-filter test_vfio`

**Version**

Output of `cloud-hypervisor --version`:
v31.0.0
It's a integration test build upon a recent master commit `41880919a0a4eb0e4fe4007337ac886128906c43`

**Logs**
```

test common_parallel::test_vfio ... FAILED

failures:

---- common_parallel::test_vfio stdout ----
mke2fs 1.45.5 (07-Jan-2020)
__populate_fs: Could not allocate block in ext2 filesystem while writing file "vmlinux"
mkfs.ext4: Could not allocate block in ext2 filesystem while populating file system

thread 'common_parallel::test_vfio' panicked at 'mkfs.ext4 command generated an error', tests/integration.rs:3768:13
stack backtrace:
   0: rust_begin_unwind
             at /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/std/src/panicking.rs:575:5
   1: core::panicking::panic_fmt
             at /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/core/src/panicking.rs:64:14
   2: integration::common_parallel::test_vfio
             at ./tests/integration.rs:3768:13
   3: integration::common_parallel::test_vfio::{{closure}}
             at ./tests/integration.rs:3730:20
   4: core::ops::function::FnOnce::call_once
             at /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/core/src/ops/function.rs:507:5
   5: core::ops::function::FnOnce::call_once
             at /rustc/d5a82bbd26e1ad8b7401f6a718a9c57c96905483/library/core/src/ops/function.rs:507:5
note: Some details are omitted, run with `RUST_BACKTRACE=full` for a verbose backtrace.


failures:
    common_parallel::test_vfio

```

**Comment by likebreath** [comment]
@michael2012z The `test_vfio` integration test has been broken for a while, but for a different reason. See: https://github.com/cloud-hypervisor/cloud-hypervisor/issues/5190#issuecomment-1431240135.

This particular failure from `mkfs.ext4` is a different issue. It is likely related to the recent upgrade of reference kernel to v6.3. The error is from writing the new kernel image `vmlinux` while generating the guest image from the provided folder.

**Comment by rmstoi** [comment]
This issue has been fixed by reducing kernel size by disabling debug configs.

**Comment by likebreath** [comment]
This issue has been fixed by #5555.
